src/settings/HRIDHendling/HRIDHandling.css

```
@@ -16,6 +16,15 @@
    padding-left: 1rem;
 }
 
+.inputLabel {
+  width: 9rem;
+  padding: 0 .5rem;
```

```suggestion
  padding: 0 0.5rem;
```
it could be more readable